test/parallel/test-net-localerror.js

```diff
@@ -20,25 +20,22 @@
 // USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 'use strict';
-require('../common');
-const assert = require('assert');
+const common = require('../common');
 const net = require('net');
 
-// Using port 0 as localPort / localAddress is already invalid.
+const connect = (opts, code, type) => common.expectsError(
+  () => net.connect(opts),
+  { code, type }
+);
```

Comment:
That's my mistake.
@pmatzavin for arrow function that span multiple lines in node's code it's prefered to use the `{}` syntax for extra clarity of the scope (even if it's just one expression like here).